cristin-project/src/main/java/no/unit/nva/cristin/projects/create/CreateCristinProjectValidator.java

```diff
@@ -61,6 +52,7 @@ private Set<ValidatedResult> validateProjectInput(NvaProject project) {
         var results = new HashSet<ValidatedResult>();
         if (isNull(project)) {
             results.add(ValidatedResult.Empty);
+            return results;
         }
         if (hasId(project)) {
```

Comment:
Should you check if it is missing an id here, or is the method name confusing me?

Reply:
Sending in an identifier is not supported, as that value is something we get from cristin, not the other way